scripts/create-package.js

```diff
@@ -40,7 +40,7 @@ fs.mkdir(newDir, { recursive: true }, err => {
     fs.writeFile(`${newSrcDir}/index.ts`, rootIndex, handleErr);
 
     fs.writeFile(
-      `${newDir}/src/${packageNamePascal}.story.tsx`,
+      `${newDir}/src/${packageNamePascal}.stories.tsx`,
```

Comment:
Is this change going to break anything in the website cc @TheSonOfThomp

Reply:
discussed with Adam - it might. Changed it back. We'll go back and rename all files to .stories.tsx at some point with a global PR